test/Instances/Attribute.spec.lua

```diff
@@ -4,7 +4,7 @@ local New = require(Package.Instances.New)
 local Attribute = require(Package.Instances.Attribute)
 local Value = require(Package.State.Value)
 
-return function()
+return function ()
```

Comment:
This space looks peculiar here and doesn't match the rest of the codebase's conventions.

Other than that though, I don't immediately see anything wrong here :)

Reply:
Oh 😆 I think that was my autocorrect, fixed!